bin/templates/project/__CLI__.xcodeproj/project.pbxproj

```diff
@@ -374,7 +374,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "cordova/lib/copy-www-build-step.sh";
+			shellScript = "node cordova/lib/copy-www-build-step.js";
```

Comment:
This change assumes `node` to be in the $PATH for this to work. I'm not familiar with OS X and I'm assuming this assumption does not cause a problem.